src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -3937,6 +3937,10 @@ void StubGenerator::generate_initial_stubs() {
   }
 
   generate_libm_stubs();
+
+  if (VM_Version::supports_avx512vlbwdq() || VM_Version::supports_fma()) {
```

Comment:
Good to check for UseAVX >= 2 as well:
if (VM_Version::supports_avx512vlbwdq() || (UseAVX >= 2) && (VM_Version::supports_fma())) {

Reply:
Works for UseAVX == 1, so added condition UseAVX >= 1.